core/logging.properties

```diff
@@ -0,0 +1,16 @@
+# log only to stdout
```

Comment:
Does this file only concern Docker? Would it then maybe make sense to name it `logging-docker.properties`?